### DISCUSSION TITLE
Multi-tenant possible?

### DISCUSSION THREAD

**Discussion by DallanQ** [gramps-project/gramps-web#152] [Q&A]
How difficult would it be to set up a multi-tenant server where people could create and invite their family to their own private trees? If that's currently difficult, is that something you'd be interested in having me work on?

**Comment by DavidMStraub** [comment]
Hi, yes this is also something I would really love to have, although it's not possible yet. I already created a [shared PostgreSQL Gramps addon](https://github.com/gramps-project/addons-source/pull/524) for this purpose. Let me convert your discussion to a Web API issue to centralize the ideas about implementing it.

**Reply by DallanQ** [reply]
Thank you! I'd be willing to help with the implementation, especially on the back end.
